e2e/tests/utils/openAppForDebugBuild.js

```diff
@@ -6,16 +6,26 @@ module.exports.openAppForDebugBuild = async function openAppForDebugBuild() {
     return;
   }
 
-  await sleep(1000);
-  await device.openURL({
-    url: process.env.EXPO_USE_UPDATES
-      // Testing latest published EAS update for the test_debug channel
-      ? getDeepLinkUrl(getLatestUpdateUrl())
-      // Local testing with packager
-      : getDeepLinkUrl(getDevLauncherPackagerUrl(platform)),
-  });
+  const deepLinkUrl = process.env.EXPO_USE_UPDATES
+    ? // Testing latest published EAS update for the test_debug channel
+      getDeepLinkUrl(getLatestUpdateUrl())
+    : // Local testing with packager
+      getDeepLinkUrl(getDevLauncherPackagerUrl(platform));
+
+  if (platform === 'ios') {
+    await device.launchApp();
+    sleep(1000);
+    await device.openURL({
+      url: deepLinkUrl,
+    });
+  } else {
+    await device.launchApp({
+      url: deepLinkUrl,
+    });
+  }
```

Comment:
See https://github.com/expo/eas-tests-example/blob/main/e2e/tests/homeScreen.e2e.js#L5
`device.launchApp` will be called twice for debug builds. I don't think this is intended.

Reply:
I've refactored the utils to launch the app correctly for both release and debug cases.